README.md

```diff
@@ -25,4 +25,4 @@ Implement the `App` with 2 pages and ability to select a person in the table.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_people-table-basics/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://GGLUTT.github.io/react_people-table-basics/) and add it to the PR description.
```

Comment:
Add the demo link to the PR description 
<img width="950" alt="image" src="https://github.com/user-attachments/assets/7da18daf-8676-4cd4-a56f-3d416f8891ae">